tests/testsuite/kubevirtresource.go

```diff
@@ -51,6 +53,29 @@ var (
 	originalKV            *v1.KubeVirt
 )
 
+/*
+TranslateBuildArch translates the build_arch to arch
+
+	case1:
+	  build_arch is crossbuild-s390x, which will be translated to s390x arch
+	case2:
+	  build_arch is s390x, which will be translated to s390x arch
+*/
+func TranslateBuildArch() string {
+	buildArch := os.Getenv("BUILD_ARCH")
+	var arch string
+
+	if buildArch != "" {
+		archElements := strings.Split(buildArch, "-")
+		if len(archElements) == 2 {
+			arch = archElements[1]
+		} else {
+			arch = archElements[0]
+		}
+	}
+	return arch
```

Comment:
Nit: this can be simplified using early return 
``` golang 
	buildArch := os.Getenv("BUILD_ARCH")
        if buildArch == {
            return ""
        }
         archElements := strings.Split(buildArch, "-")
	if len(archElements) == 2 {
		return  archElements[1]
	} else {
		return archElements[0]
	}
```

Reply:
done